src/style.css

```diff
@@ -0,0 +1,69 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.conteiner {
+  position: relative;
+}
+
+.stars {
+  display: flex;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star:not(:first-child) {
+  content: "";
+  background-image: url(./images/star.svg);
+}
+
+.stars--0 .stars__star {
```

Comment:
```suggestion
.stars--0 .stars__star,
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:first-child,
... {
    background-image: url(./images/star-active.svg);
}
```

Reply:
.stars--00 .stars__star has ./images/star.svg (not images/star-active.svg)